### ISSUE TITLE
Sell orders placed above manual center price

### ISSUE THREAD

**Issue by cloud-8** [Codaone/DEXBot#522]
## Expected Behavior

Sell orders will not be placed above manual center price. 

## Actual Behavior

Sell order placed above manual center. 

## Steps to Reproduce the Problem

The easiest way to replicate is try it on an immature UIA to UIA market which is the same asset. E.g. sparkdex.BTC/open.BTC. These markets have organic orderbook however which might change calculations for center price so better to try in a market without any orders. However for like-to-like asset pairs like BTC to BTC, center price should always be 1. 

Following Params: 

Mode: Valley
Spread: 1.5%
Increment 1%
Market centre price: no
Manual center price: 1
Lower bound: 0.85
Upper bound: 1.15
Allow instant fill: yes

## Specifications

  - Version: 0.95 Dexbot
  - OS: Ubuntu 18.04

**Comment by bitfag** [comment]
For staggered orders, manual center price setting is only used at initial orders placement stage. So, if you had several orders filled at the buy side, the same number of new orders will be placed on a sell side to maintain target spread.

**Comment by bitfag** [comment]
For xxx.BTC/yyy.BTC markets I would recommend to use Relative Orders with fixed center price and maybe Dynamic Spread feature (one caveat: there is no "min spread" limiter for Dynamic Spread to prevent spread bee too close).

**Comment by cloud-8** [comment]
> For staggered orders, manual center price setting is only used at initial orders placement stage. So, if you had several orders filled at the buy side, the same number of new orders will be placed on a sell side to maintain target spread.

So in my case spread is 0.75% either side of 1 during initial order placement for a total of 1.5% spread. Lets say alot of my orders get taken on the bid, increasing the spread, so Dexbot would place new orders to maintain the spread, however if none of my orders have also been taken on the ask, then how can my bid order placement drift upwards ? I have 0.75%  at least as a buffer. If the Asks are in the exact same location as previously, then dexbot should just be refilling orders to where they were taken? 

Relative orders is only 1 order either side, to make a nice market I want to use Valley mode which is more natural for people to engage with and see there is varying liquidity.

**Comment by bitfag** [comment]
Can you provide a logfile (dexbot.log) and bitshares account name to allow us to investigate what exactly happened?

**Comment by MarkoPaasila** [comment]
@cloud-8 are you using _center price offset based on asset balances_? That would adjust orders up or down when orders fill. But even then it shouldn't exceed center price.

**Comment by bitfag** [comment]
@MarkoPaasila he uses Staggered Orders.

**Comment by MarkoPaasila** [comment]
So the problem is that the user might assume that manual center price means that it remains fixed (in Staggered Orders). Maybe we should change the term/description to better describe its purpose. Here's some suggestions:
placeholder centerprice,
initial centerprice, 
starting price, 
assumed price, 
bootstrapping price

**Comment by PermieBTS** [comment]
Starting price is better

**Comment by cloud-8** [comment]
> @cloud-8 are you using _center price offset based on asset balances_? That would adjust orders up or down when orders fill. But even then it shouldn't exceed center price.

Yes I am, however it still shouldnt IMO be placing above center. You can see this now in sparkdex.BTC/open.BTC market. Dexbot is placing orders below 1 on the Ask.
